[revealjs] Mention `title-fancy/title-slide.html` as the partial starter to use for the title slide. (#1265)
quarto-dev/quarto-web: docs/journals/templates.qmd

title-slide.html

> :   HTML used for the presentation title slide. Quarto uses by default a fancier `title-slide.html` partial leveraging [Authors & Affiliations](/docs/journals/authors.qmd) handling that should be used as a starter if you want to tweak this partials. See [source code here](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/revealjs/pandoc/title-fancy/title-slide.html).

Reviewer: ```suggestion
:   HTML used for the presentation title slide. By default, Quarto uses a fancy `title-slide.html` partial that leverages [Authors & Affiliations](/docs/journals/authors.qmd) handling. Use the `title-slide.html` [source code](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/revealjs/pandoc/title-fancy/title-slide.html) as a starter if you want to tweak this partial.
```
